api/export_api.py

```
@@ -114,7 +114,7 @@ def export_chats():
         rules = current_app.config.get("EXCLUSION_RULES") or []
 
         # ── Database reading via service layer ────────────────────────────────
-        with _open_global_db(workspace_path) as (global_db, global_db_path):
+        with open_global_db(workspace_path) as (global_db, global_db_path):
```

Ruff (0.15.14)

[warning] Unpacked variable `global_db_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)